public/pages/ThreatIntel/containers/AddThreatIntelSource/AddThreatIntelSource.tsx

```diff
@@ -385,12 +578,20 @@ export const AddThreatIntelSource: React.FC<AddThreatIntelSourceProps> = ({
         <EuiText>
           <h4>Types of malicious indicators</h4>
         </EuiText>
-        <EuiSpacer />
-        <EuiCheckboxGroup
-          options={checkboxes}
-          idToSelectedMap={checkboxIdToSelectedMap}
-          onChange={onIocTypesChange}
-        />
+        <EuiText color="subdued" size="s">
+          <p>
+            Select atleast one IOC type to select from the{' '}
```

Comment:
Minor typo:
No space between "at" and "least".

Also, I've seen "IoC" used elsewhere in the UI. Is there a preferred capitalization pattern?

Reply:
Changing all IOC to IoC